receiver/aerospikereceiver/metadata.yaml

```
@@ -0,0 +1,136 @@
+name: aerospikereceiver
+
+resource_attributes:
+  node_name:
+    description: Name of the Aerospike node collected from
+    type: string
+  aerospike.namespace:
+    description: Name of the Aerospike namespace
+    type: string
+
+attributes:
+  namespace_component:
+    value: component
+    description: Individual component of a namespace
+    enum:
+      - data
+      - index
+      - set_index
+      - sindex
+  scan_type:
+    value: type
+    description: Type of scan operation performed on a namespace
+    enum:
+      - aggr
+      - basic
+      - ops_bg
+      - udf_bg
+  scan_result:
+    value: result
+    description: Result of a scan operation performed on a namespace
+    enum:
+      - abort
+      - complete
+      - error
+  transaction_type:
+    value: type
+    description: Type of transaction performed on a namespace
+    enum:
+      - delete
+      - read
+      - udf
+      - write
+  transaction_result:
+    value: result
+    description: Result of a transaction performed on a namespace
+    enum:
+      - error
+      - filtered_out
+      - not_found
+      - success
+      - timeout
+  connection_type:
+    value: type
+    description: Type of connection to an Aerospike node
+    enum:
+      - client
+      - fabric
+      - heartbeat
+  connection_op:
+    value: op
```

```suggestion
    value: operation
```
Unless "op" is specifically used in aerospike-lingo, this would typically not be abbreviated.